<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[The URL opened is https://www.goindigo.in/?linkNav=home_header]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the value at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-src]]>
    </line>
    <line>
      <![CDATA[Entered the value at Locator  name:or-dest]]>
    </line>
    <line>
      <![CDATA[Entered the Key at Locator  name:or-dest]]>
    </line>
    <line>
      <![CDATA[TC01 Failed]]>
    </line>
    <line>
      <![CDATA[TC02 Failed]]>
    </line>
    <line>
      <![CDATA[TC03 Failed]]>
    </line>
    <line>
      <![CDATA[Hovered the value at Locator  xpath://*[@id='navbarSupportedContent']/ul/li[2]/a]]>
    </line>
    <line>
      <![CDATA[Hovered and clicked the value at Locator  linktext:Lounge Services]]>
    </line>
    <line>
      <![CDATA[Entered the value at Locator  xpath://a[starts-with(@class,'ac_closed acc-lnk panel-title collapsed counterdisplay')]]]>
    </line>
    <line>
      <![CDATA[Closed the browser  ]]>
    </line>
  </reporter-output>
  <suite started-at="2019-09-14T13:28:43Z" name="Suite" finished-at="2019-09-14T13:29:12Z" duration-ms="29485">
    <groups>
    </groups>
    <test started-at="2019-09-14T13:28:43Z" name="Test" finished-at="2019-09-14T13:29:12Z" duration-ms="29485">
      <class name="mainScript.DriverScript">
        <test-method signature="tc01()[pri:1, instance:mainScript.DriverScript@6bbe2511]" started-at="2019-09-14T13:28:43Z" name="tc01" finished-at="2019-09-14T13:29:05Z" duration-ms="22062" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at mainScript.DriverScript.tc01(DriverScript.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[The URL opened is https://www.goindigo.in/?linkNav=home_header]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the value at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-src]]>
            </line>
            <line>
              <![CDATA[Entered the value at Locator  name:or-dest]]>
            </line>
            <line>
              <![CDATA[Entered the Key at Locator  name:or-dest]]>
            </line>
            <line>
              <![CDATA[TC01 Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- tc01 -->
        <test-method signature="tc02()[pri:2, instance:mainScript.DriverScript@6bbe2511]" started-at="2019-09-14T13:29:05Z" name="tc02" finished-at="2019-09-14T13:29:05Z" duration-ms="251" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at mainScript.DriverScript.tc02(DriverScript.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[TC02 Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- tc02 -->
        <test-method signature="tc03()[pri:3, instance:mainScript.DriverScript@6bbe2511]" started-at="2019-09-14T13:29:05Z" name="tc03" finished-at="2019-09-14T13:29:06Z" duration-ms="1159" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at mainScript.DriverScript.tc03(DriverScript.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[TC03 Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- tc03 -->
        <test-method signature="tc04()[pri:4, instance:mainScript.DriverScript@6bbe2511]" started-at="2019-09-14T13:29:06Z" name="tc04" finished-at="2019-09-14T13:29:12Z" duration-ms="5984" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Hovered the value at Locator  xpath://*[@id='navbarSupportedContent']/ul/li[2]/a]]>
            </line>
            <line>
              <![CDATA[Hovered and clicked the value at Locator  linktext:Lounge Services]]>
            </line>
            <line>
              <![CDATA[Entered the value at Locator  xpath://a[starts-with(@class,'ac_closed acc-lnk panel-title collapsed counterdisplay')]]]>
            </line>
            <line>
              <![CDATA[Closed the browser  ]]>
            </line>
          </reporter-output>
        </test-method> <!-- tc04 -->
      </class> <!-- mainScript.DriverScript -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
